chore: DLT-1809 add blog post for new tokens page and empty state

ninamarina opened the pull request:
# Add blog post for new tokens page and empty state

## :hammer_and_wrench: Type Of Change

<!--- Tick or place an `x` for the type of change. Should match the type in the commit message / PR title
in https://github.com/dialpad/dialtone/blob/staging/.github/COMMIT_CONVENTION.md -->

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Other (chore)

## :book: Jira Ticket
[DLT-1809]
<!--- Enter the URL of the Jira ticket associated with this PR -->

## :book: Description
Adds blog post to show the new search in the tokens page and the Empty State component: https://dialtone.dialpad.com/deploy-previews/pr-396/about/whats-new/posts/2024-7-8.html
<!--- Describe specifically what the changes are -->


[DLT-1809]: https://dialpad.atlassian.net/browse/DLT-1809?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

ninamarina committed:
add blogpost for tokens page and empty state

braddialpad reviewed (commented):
Super weird... why is the preview not in the correct order.

Was this working fine locally?

![Screenshot 2024-07-04 at 11 41 52 AM](https://github.com/dialpad/dialtone/assets/64808812/2c1315c1-7b4f-4a09-8e45-f4bb4c893049)

braddialpad commented:
> Super weird... why is the preview not in the correct order.
> 
> Was this working fine locally?

nm I was confused because the empty state image was not in the empty state section

braddialpad commented on apps/dialtone-documentation/docs/about/whats-new/posts/2024-7-5.md:
```diff
@@ -0,0 +1,58 @@
+
+* We are rolling out a new component, the [Empty State Component](/components/empty-state.html).
+
+![Empty State example](/assets/images/empty-state.png)
```
We can summarize the updates here via text, but the images should be moved to their respective sections.

ninamarina replied:
I just wanted to show an image in the first 10 lines, to make sure people see it in the email. Maybe I can move the token gif to the start of the tokens section and at least show that one.

ninamarina replied:
I ended up removing the image for the empty state, because it looked like something was really missing

braddialpad commented on apps/dialtone-documentation/docs/about/whats-new/posts/2024-7-5.md:
```diff
@@ -0,0 +1,58 @@
+
+Change theme and format in the topbar. Available formats are: **CSS**, **Android** and **iOS**. Change the theme to see how the token would look both in light and dark mode. Use the "Share filter" button to get a URL with the search and filters applied.
+
+![Token Filters](/assets/images/tokens-filters.png)
```
Let's also mention how you can hover composition tokens to see their values, since that might not be obvious.

ninamarina committed:
fix to move images to sections

braddialpad commented on apps/dialtone-documentation/docs/about/whats-new/posts/2024-7-5.md:
```diff
@@ -0,0 +1,62 @@
+
+Ever needed to show a "No Results" page? Or an empty call history?
+
+For these a many more scenarios the Empty State Component is now available. You can checkout all the variants and examples in the [Empty State Component Page](/components/empty-state.html).
```
"For these and"

braddialpad commented on apps/dialtone-documentation/docs/about/whats-new/posts/2024-7-5.md:
```diff
@@ -0,0 +1,62 @@
+---
+heading: New search in the Tokens Page and new Empty State Component
```
Can just captialize the first letter of the heading, based on our [writing guidlines ](https://dialtone.dialpad.com/deploy-previews/pr-396/guides/writing-guidelines/#capitalization)

braddialpad reviewed (approved):
Couple small typo things. Looks good!

ninamarina committed:
fix typos

ninamarina committed:
improve writing

ninamarina committed:
add examples to empty state

iropolo reviewed (approved):
Great!!

iropolo commented:
The gif at the top confuses a little bit because seems like it is not a gif at the beginning.

This is probably something that we will deal repeatedly with gifs and images in our blogs...
Could we add something to make it clear and easily to read that there is an image/gif? maybe a box-shadow, some border or background... or even make it a little bit more smaller.

I saw @fede-dp add this border `d-ba d-bc-subtle` in the empty-state component section.

ninamarina committed:
add border to gif

ninamarina committed:
test email

ninamarina committed:
replace images

ninamarina committed:
add empty state image and change date

ninamarina committed:
Revert "test email"

This reverts commit 2fc0552c7ffdf98541b354c94c9441df9699fded.

ninamarina committed:
fix date

ninamarina commented:
> The gif at the top confuses a little bit because seems like it is not a gif at the beginning.
> 
> This is probably something that we will deal repeatedly with gifs and images in our blogs... Could we add something to make it clear and easily to read that there is an image/gif? maybe a box-shadow, some border or background... or even make it a little bit more smaller.
> 
> I saw @fede-dp add this border `d-ba d-bc-subtle` in the empty-state component section.

I added a box shadow to all images, I think it's better now. Also replaced the gif for a static image, I couldn't add the border to the gif, and also I don't think it added much value anyways.

iropolo commented:
Looks better now!!
